tests/test_arithmetic.py

```diff
@@ -0,0 +1,90 @@
+"""
+"""
+import pytest
+
+import calc
+
+
+def test_parse_subtraction():
+    input_text = "5-3"
+    calculator = calc.Calc(text=input_text)
+    assert calculator.parse() == 2
+
+
+def test_parse_multiplication_identity():
```

Comment:
I'm weird and I appreciate symmetry. Do you have objections to adding the missing identity functions?